README.md

```diff
@@ -28,15 +37,48 @@ import yt_idefix
 
 Now `yt.load` will be able to read Pluto/Idefix output files.
 
-## Supported formats
+### Dataset's Metadata
+The metadata is automaticly parsed from data file, definitions header file and inifile when loading dataset.
```

Comment:
Since auto-discovery for inifile and definitions header may not succeed, let's rephrase this
```suggestion
The metadata is parsed from data file, definitions header file and inifile when loading dataset.
```